num_enum/tests/try_build/compile_fail/multiple_crate_path.stderr

```diff
@@ -0,0 +1,5 @@
+error: Expected exactly one `crate` argument
```

Comment:
I'd expect this error message to be "at most one" rather than "exactly one" - it's allowed to not specify a `crate` argument.